src/Mesh/Grids.jl

```diff
@@ -231,4 +231,134 @@ function computegeometry(topology::AbstractTopology{dim}, D, ξ, ω, meshwarp,
 end
 # }}}
 
+
+# {{{ compute element size for box domain 
+function compute_element_size(dim, Nq, vgeo, e) 
```

Comment:
Distances are related to the grid. Where would you like them to be put instead of grids.jl? I can move them there. They are not LES specific so that I can't put them in the LES calculations.

Reply:
Yes, but this code only works in the specific case of an affine element that is aligned with the coordinate axes. That is this will not work on the sphere or a warped mesh.

At a minimum is should error out if
```
x[1, :, :] != x[1, 1, 1]
x[2, :, :] != x[2, 1, 1]

y[:, 1, :] != y[1, 1, 1]
y[:, 2, :] != y[1, 2, 1]

z[:, :, 1] != z[1, 1, 1]
z[:, :, 2] != z[1, 1, 2]
```
(or something along those lines)

Reply:
I'm not opposed to a `compute_element_size` function. I'd just like something that works in all cases, or errors out gracefully when it does not work.